tests/ord-service/tests/api_test.go

```diff
@@ -357,6 +357,22 @@ func TestORDService(stdT *testing.T) {
 			require.Equal(t, testData.appInput.Name, gjson.Get(respBody, "value.0.title").String())
 			require.Equal(t, *testData.appInput.Description, gjson.Get(respBody, "value.0.description").String())
 		})
+		t.Run(fmt.Sprintf("Requesting System Instances with apis for tenant %s returns them as expected", testData.msg), func(t *testing.T) {
+			respBody := makeRequestWithHeaders(t, testData.client, testData.url+"/systemInstances?$format=json&$expand=apis", testData.headers)
```

Comment:
It's more conventional to be:

`/systemInstances?$expand=apis&$format=json`

Same below for the events.